src/parser/jobs/pld/modules/FightOrFlight.tsx

```diff
@@ -60,18 +62,42 @@ export class FightOrFlight extends BuffWindow {
 
 		this.addEvaluator(new ExpectedActionsEvaluator({
 			expectedActions: [
-				{action: this.data.actions.GORING_BLADE, expectedPerWindow: 1},
-				{action: this.data.actions.CONFITEOR, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_FAITH, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
-				{action: this.data.actions.BLADE_OF_VALOR, expectedPerWindow: 1},
-				{action: this.data.actions.HOLY_SPIRIT, expectedPerWindow: 1},
+				{action: actions.GORING_BLADE, expectedPerWindow: 1},
+				{action: actions.CONFITEOR, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_FAITH, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
+				{action: actions.BLADE_OF_VALOR, expectedPerWindow: 1},
 			],
-			suggestionIcon: this.data.actions.FIGHT_OR_FLIGHT.icon,
+			suggestionIcon: actions.FIGHT_OR_FLIGHT.icon,
+			suggestionContent: <Trans id="pld.fightorflight.suggestions.gcd_actions.content">
+				Try to land at least one cast of <DataLink action="GORING_BLADE" />
+				, <DataLink action="CONFITEOR" />, <DataLink action="BLADE_OF_FAITH" />, <DataLink action="BLADE_OF_TRUTH" />
+				, <DataLink action="BLADE_OF_VALOR" />, and the highest potency combination of <DataLink action="SEPULCHRE" />, <DataLink status="DIVINE_MIGHT" />
+				empowered <DataLink action="HOLY_SPIRIT" />, <DataLink action="SUPPLICATION" />, <DataLink action="ATONEMENT" /> and <DataLink action="ROYAL_AUTHORITY" />
+				during every <DataLink action="FIGHT_OR_FLIGHT" /> window.
+			</Trans>,
+			suggestionWindowName,
+			severityTiers: SEVERITIES.MISSED_ACTIONS,
+		}))
+
+		this.addEvaluator(new ExpectedActionGroupsEvaluator({
+			expectedActionGroups: [{
+				actions: [
+					actions.ROYAL_AUTHORITY,
+					actions.ATONEMENT,
+					actions.SUPPLICATION,
+					actions.SEPULCHRE,
+					actions.HOLY_SPIRIT,
+				],
+				overrideHeader: actions.SEPULCHRE,
+				expectedPerWindow: 3,
+			}],
+			suggestionIcon: actions.FIGHT_OR_FLIGHT.icon,
 			suggestionContent: <Trans id="pld.fightorflight.suggestions.gcd_actions.content">
```

Comment:
_Action:_ Same comment here as above; a simpler rewrite (also reordered for the "combo" order) could look like:

`Try to land at least three of <DataLink action="ROYAL_AUTHORITY" />, <DataLink action="ATONEMENT" />, <DataLink action="SUPPLICATION" />, <DataLink action="SEPULCHRE" />, and <DataLink status="DIVINE_MIGHT" /> empowered <DataLink action="HOLY_SPIRIT" /> during every <DataLink action="FIGHT_OR_FLIGHT" /> window.`

Given that the suggestion is only looking for 3 and not a specific 3, you can drop the "highest combination of" qualifier as well.

Reply:
Adjusted accordingly, though I've also cut the "at least" and just said "Try to land three of", since the remaining usages are already supposed to be taken with the confiteor combo and goring blade.